mmdet/core/hook/yolox_mode_switch_hook.py

```diff
@@ -27,8 +27,7 @@ def before_train_epoch(self, runner):
             model = model.module
         if (epoch + 1) == runner.max_epochs - self.num_last_epochs:
             runner.logger.info('No mosaic and mixup aug now!')
-            # TODO
-            train_loader.dataset.enable_mosaic = False
-            train_loader.dataset.enable_mixup = False
+            train_loader.dataset.update_skip_type_keys(
```

Comment:
Making `['Mosaic', 'RandomAffine', 'MixUp']` as argumens of `_init_` may be better

Reply:
ok